src/components/EditCoursePage/CollapsibleCourseRunFields.jsx

```diff
@@ -5,13 +5,15 @@ import { Field, FieldArray, submit } from 'redux-form';
 import { Collapsible } from '@edx/paragon';
 import { connect } from 'react-redux';
 
+import Pill from '../Pill';
```

Comment:
Unrelated to your perfect change here - but the fact that our style linter doesn't enforce alphabetical imports is driving me batty. Look at this list! It's nonsense.

Reply:
Starting a conversation in the fedx channel about this